docs/concepts/configuration/taint-and-toleration.md

```diff
@@ -273,6 +274,7 @@ To make sure that turning on this feature doesn't break DaemonSets, starting in
   * `node.kubernetes.io/memory-pressure`
   * `node.kubernetes.io/disk-pressure`
   * `node.kubernetes.io/out-of-disk` (*only for critical pods*)
+  * `node.kubernetes.io/unschedulable`
```

Comment:
This specific taint & toleration is only added in 1.9, but their previous paragraph says it's added in 1.6 & 1.8.

Reply:
Append version number :)